closed/CopyToBuildJdk.gmk

```diff
@@ -216,12 +216,7 @@ ifeq (true,$(OPENJ9_ENABLE_DDR))
   $(OPENJ9_VM_BUILD_DIR)/j9ddr.dat : run-ddrgen
 
   run-ddrgen :
-    ifeq (true,$(OPENJ9_ENABLE_CMAKE))
 	  $(EXPORT_MSVS_ENV_VARS) $(MAKE) -C $(OPENJ9_VM_BUILD_DIR) j9ddr
```

Comment:
Could you please trim the leading spaces from this line (that wouldn't be here if it were never conditional)?